lms/templates/instructor/instructor_dashboard_2/cohort-form.underscore

```diff
@@ -25,7 +25,7 @@
                             <%- gettext('Cohort Name') %> *
                             <span class="sr"><%- gettext('(Required Field)')%></span>
                     </label>
-                    <input name="cohort-name" value="<%= cohort_name_value %>" class="input cohort-name" id="cohort-name"
+                    <input name="cohort-name" value="<%- cohort_name_value %>" class="input cohort-name" id="cohort-name"
```

Comment:
@efischer19 I think you are just confirming here that it is ok and appropriate to escape value attributes in an HTML page.  Is that accurate?  Thanks.